histomicstk/segmentation/nuclear/max_clustering.py

```diff
@@ -74,7 +74,7 @@ def max_clustering(im_response, im_fgnd_mask, r=10):
     # compute object properties
     obj_props = skimage.measure.regionprops(im_label, im_response_nmzd)
 
-    obj_props = [prop for prop in obj_props if np.isfinite(prop.weighted_centroid).all()]
+    obj_props = [prop for prop in obj_props if np.isfinite(prop['centroid_weighted'].all()]
```

Comment:
You've dropped a parenthesis -- `np.isfinite(prop['centroid_weighted'].all()` should be `np.isfinite(prop['centroid_weighted']).all()`.